api/agent.go

```diff
@@ -365,17 +365,26 @@ func NewControllerWrapper(config *loadtest.Config, controllerConfig interface{},
 		return nil, err
 	}
 
+	modAdmins := 0
+	if config.UsersConfiguration.PercentOfUsersAreAdmin > 0 {
+		modAdmins = int(1 / config.UsersConfiguration.PercentOfUsersAreAdmin)
+	}
+
 	return func(id int, status chan<- control.UserStatus) (control.UserController, error) {
 		id += userOffset
 
 		username := fmt.Sprintf("%s-%d", namePrefix, id)
 		email := fmt.Sprintf("%s-%d@example.com", namePrefix, id)
 		password := "testPass123$"
 
-		// If UsersFilePath was set, and we haven't yet consumed all of the credentials
-		// provided there, ovewrite this user's credentials with the next available
-		// user in that file
-		if len(creds) > 0 && id < len(creds) {
+		if modAdmins > 0 && id%modAdmins == 0 {
```

Comment:
One caveat with this approach is that we'd always be running admins first and always using the same user. Not a big deal I suppose but something to note.